typing_extensions/src_py3/typing_extensions.py

```diff
@@ -363,21 +368,20 @@ def __getitem__(self, parameters):
                                       '{} accepts only single type'.format(self._name))
             return _GenericAlias(self, (item,))
 
-    Final = _FinalForm('Final', doc=
-        """A special typing construct to indicate that a name
-        cannot be re-assigned or overridden in a subclass.
-        For example:
+    Final = _FinalForm('Final',
+                       doc="""A special typing construct to indicate that a name
+                       cannot be re-assigned or overridden in a subclass.
```

Comment:
A style similar is what I'm recommending for `Annotated` on https://github.com/python/typing/issues/600

Reply:
(This isn't really a code review comment, it's a cross reference, so resolving -- the cross reference will still be in #666.)